src/jit/importer.cpp

```diff
@@ -7045,19 +7045,17 @@ GenTreePtr                Compiler::impFixupStructReturn(GenTreePtr     call,
     call->gtCall.gtReturnType = call->gtType;
 
     // Get the classification for the struct.
-    SYSTEMV_AMD64_CORINFO_STRUCT_REG_PASSING_DESCRIPTOR structDesc;
-    eeGetSystemVAmd64PassStructInRegisterDescriptor(retClsHnd, &structDesc);
-    if (structDesc.passedInRegisters)
+    GenTreeCall* callPtr = call->AsCall();
```

Comment:
This is a minor point, but my preference would be to avoid the use of "Ptr". Having "call" and "callPtr" might lead one to believe that callPtr is a "parent Ptr" (in the often-used paradigm of using a parent pointer to a child, that can be updated if the node is changed). If you want to avoid using "gtCall." or "AsCall()" how about "callNode"?